pages/en/android-environment/deploying-apps.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Deploying Android apps to Chromebooks
+title: ADB and testing Android apps
```

Comment:
If the title is being changed, should the file name be edited to match? Or will that result in broken links and whatnot.

Reply:
That totally makes sense!